packages/flutter_tools/test/commands/create_test.dart

```diff
@@ -286,10 +286,10 @@ void main() {
   testUsingContext('plugin project with custom org', () async {
     return _createProject(
       projectDir,
-      <String>['--no-pub', '--template=plugin', '--org', 'com.bar.foo'],
+      <String>['--no-pub', '--template=plugin', '--org', 'com.bar.foo', '--project', 'xyz'],
```

Comment:
This is an OK test, but instead of putting it into "plugin project with custom org", could you make a new test case, called "setting custom project name" (or something similar)? Also, it would be good to also try a name that is not allowed (e.g. "x.y.z") and make sure that it throws a tool exit (see examples elsewhere in the file).